kyuubi-server/src/main/scala/org/apache/kyuubi/session/FileSessionConfAdvisor.scala

```diff
@@ -44,9 +48,12 @@ class FileSessionConfAdvisor extends SessionConfAdvisor {
 }
 
 object FileSessionConfAdvisor extends Logging {
-  private val sessionConfCache: LoadingCache[String, JMap[String, String]] =
+  private var reloadDuration: Long = _
```

Comment:
the reloadDuration seems need be initialized for one time.

We should not get it from the SessionConf, which is dynamic.

Reply:
It is tricky now, it depends on the first user session conf.
It is non-deterministic.

Reply:
how about just get it from `KyuubiConf()`?

Reply:
nice catch ，I will change it